Update to use the latest version of cisagov/guacamole-composition

jsf9k opened the pull request:
## 🗣 Description ##

This pull request updates the Ansible role to use the latest version of [cisagov/guacamole-composition](https://github.com/cisagov/guacamole-composition), which in turn uses the latest releases (i.e., the 1.5 releases) of the [guacamole/guacamole](https://hub.docker.com/r/guacamole/guacamole) and [guacamole/guacd](https://hub.docker.com/r/guacamole/guacd) Docker containers.

## 💭 Motivation and context ##

See cisagov/guacamole-composition#58.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

jsf9k committed:
Update to the latest prerelease of cisagov/guacamole-composition

jsf9k committed:
Pull the latest releases of guacamole/guacd and guacamole/guacamole

These are the releases used by the latest prerelease of
cisagov/guacamole-composition, and all releases of
cisagov/guacamole-composition going forward.

jsf9k committed:
Use the latest prerelease of cisagov/guacamole-composition

jsf9k committed:
Use the latest prerelease of cisagov/guacamole-composition

jsf9k committed:
Pull the latest releases of guacamole/guacd and guacamole/guacamole

These are the releases used by the latest prerelease of
cisagov/guacamole-composition.

jsf9k committed:
Use the latest prerelease of cisagov/guacamole-composition

jsf9k committed:
Update Molecule tests for new PostgreSQL Docker image

jsf9k committed:
Use the latest release of cisagov/guacamole-composition

dav3r reviewed (approved):
🎉

mcdonnnj commented on tasks/main.yml:
```diff
@@ -41,15 +41,9 @@
-        # See cisagov/guacamole-composition#37 for more information as
-        # to when this limitation can be remedied.
-        - postgres:13
+        - guacamole/guacd:1.5.1
+        - guacamole/guacamole:1.5.1
```
Take it or leave it but I missed that these weren't sorted previously.

```suggestion
        - guacamole/guacamole:1.5.1
        - guacamole/guacd:1.5.1
```

jsf9k committed:
Alphabetize Docker image names

Co-authored-by: Nick <50747025+mcdonnnj@users.noreply.github.com>

mcdonnnj committed:
Update the `molecule` requirement

The molecule plugins have been consolidated from their own molecule-*
packages into a monolithic molecule-plugins package. With the release
of molecule v5 we have run afoul of this change and must update our
requirements accordingly.
